### DISCUSSION TITLE
change .env path for prisma database_url??

### DISCUSSION THREAD

**Discussion by nyctonio** [prisma/prisma#15976] [Q&A] [answered]
I am using turborepo and setup prisma as a package but i want to make .env file in the root folder how to link the .env file with the prisma package

**Comment by nurul3101** [comment, marked as answer]
Hey @nyctonio 👋 

Prisma should be able to detect the env file in the root folder without any additional configuration, are you getting any specific error?
Here's the [reference](https://www.prisma.io/docs/guides/development-environment/environment-variables#env-file-locations) on which locations does prisma look for finding env file. Also, here's an example repository of [Turborepo with Prisma](https://github.com/aiji42/turbo-with-prisma) which should be helpful to you.

Do let me know in case you are still facing an issue.

**Reply by vladi-stevanovic** [reply]
:wave: Hello @nyctonio , I’m going through pending GitHub Discussions to check if you need any help. Is this issue still open?

Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
